### ISSUE TITLE
Inductor & LED slow the app

### ISSUE THREAD

**Issue by 64-Tesseract** [Proto-App/Proto-Android#35]
I hooked up an LED, an inductor, and a DC source in a loop, and after the LED blew, the app slowed to a crawl until I fixed the LED. I think the inductor's remaining charge isn't handled efficiently, because when I removed the inductor from the loop the LED blew without the framerate dropping.

**Comment by Proto-App** [comment]
@64-Tesseract 
Yes you are right that voltage value causes issue in this case. 

Our inductor component has ideal implementation, it means that it has no series resistance, capacitance and we don't want to fix our engine to handle this case because it behaves ok. You can make equivalent circuit of an real inductor and energy will be discharged to ground: 

![inductor](https://user-images.githubusercontent.com/63373346/117478584-a5011e80-af5f-11eb-904c-a8790a00c170.png)

Later on we will try to implement switch between real and ideal  implementation for inductor/capacitor.
